django_saml2_auth/saml.py

```diff
@@ -156,6 +156,13 @@ def get_metadata(
             )
 
 
+def get_custom_acs_uri() -> str:
+    get_custom_acs_url_hook = dictor(settings.SAML2_AUTH, "TRIGGER.GET_CUSTOM_ASSERTION_URI")
+    if get_custom_acs_url_hook:
+        return run_hook(get_custom_acs_url_hook)
+    return dictor(settings.SAML2_AUTH, "CUSTOM_ASSERTION_URI")
```

Comment:
1. The `CUSTOM_ASSERTION_URI` looks redundant. If the hook exists and returns a valid URL (as string), then we're good to go. Otherwise we should use the `ASSERTION_URL`.
2. It feels like the hook function can be integrated into the `get_assertion_url` [function](https://github.com/grafana/django-saml2-auth/blob/df32f5f05a8261c1cf00f74aedbaf696c4383bda/django_saml2_auth/saml.py#L34). WDYT?

Reply:
Yeah I agree with 1st. If you need static URL, you can just return it (with lambda or whatever).
Will remove the `CUSTOM_ASSERTION_URI`

Reply:
For the 2nd, it's used to get the domain, which then is used in a few other places, so it's a bit scary, not to break anything

Reply:
No worries! I'm always here to help.

Reply:
I removed `CUSTOM_ASSERTION_URI`!

Reply:
We still need the `domain` for fetching the metadata, so changing `get_assertion_url` is not a good idea imo